experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -74,147 +74,153 @@ include("slab/slab_init.jl")
 include("slab_ocean/slab_init.jl")
 include("slab_ice/slab_init.jl")
 
-if land_sim == "slab"
-    slab_sim = slab_init(FT; tspan, dt = Δt_cpl, space = boundary_space, saveat = saveat, mask = mask)
+if land_sim_name == "slab"
+    land_sim = slab_init(FT; tspan, dt = Δt_cpl, space = boundary_space, saveat = saveat, mask = mask)
 end
-if land_sim == "bucket"
-    slab_sim = bucket_init(FT, FT.(tspan); dt = FT(Δt_cpl), space = boundary_space, saveat = FT(saveat))
+if land_sim_name == "bucket"
+    land_sim = bucket_init(FT, FT.(tspan); dt = FT(Δt_cpl), space = boundary_space, saveat = FT(saveat))
 end
 
+prescribed_sst = true
```

Comment:
should be read in from command line - this overwrites it. I think this is why the buildkite plot isnt being made.

Reply:
Also, it seems like prescribed_sst now flips between aquaplanet and AMIP. Should we be passing that in, instead? i.e. pass in a CLI called mode which is either amip or aquaplanet, rather than pass in prescribed_sst? Because prescribed_sst also affects what ice is doing. maybe a bit confusing as is.

Reply:
Oops! This line shouldn't be there (it was leftover from a test - nice catch!). As for changing this flag to `prescribed_sst` -> `mode` (or something like that), that can be done. :)